File: CreationalPatterns/Builder/CSharp/Car.cs
namespace Builder
{
    public class Car
    {
        public int Seats { get; set; }
        public string Engine { get; set; }
        public bool TripComputer { get; set; }
        public bool GPS { get; set; }

        public override string ToString()
        {
            return $"Seats: {Seats}, Engine: {Engine}, TripComputer: {TripComputer}, GPS: {GPS}";
        }
    }
}

File: CreationalPatterns/Builder/TypeScript/Car.ts
export class Car {
    seats: number = 0;
    engine: string = '';
    tripComputer: boolean = false;
    gps: boolean = false;

    toString(): string {
        return `Seats: ${this.seats}, Engine: ${this.engine}, TripComputer: ${this.tripComputer}, GPS: ${this.gps}`;
    }
}

